docs/pages/includes/database-access/db-audit-events.mdx

```diff
@@ -0,0 +1,38 @@
+Database audit events will appear in the Audit Log including database sessions starting,
```

Comment:
I think the first sentence could be more clear. Or maybe just remove it.

Reply:
Please see updates